packages/client/hmi-client/src/types/common.ts

```diff
@@ -20,6 +20,11 @@ export interface FeatureConfig {
 	isPreview: boolean;
 }
 
+export enum DrillDownTabs {
+	Wizard = 'Wizard',
+	Notebook = 'Notebook'
+}
```

Comment:
Every drilldown seems to recreate this `enum` with a different name but the same values. Is there a reason for this, or can they all use a common `enum`?

Reply:
opened issue

Reply:
Yeah it'd be better to use the same one since they all just have those two tabs. @blanchco is this fine?